### PR TITLE
Add functions for running `mix test` and `mix test.watch` in a child app

### PR DESCRIPTION
**Pull request by frahugo**

### CODE REVIEW

File: civilcode.shrc [hunk 1]
```diff
@@ -4,9 +4,19 @@ alias dea="docker-compose exec application"
...
 alias deam="docker-compose exec application mix"
 alias deatm="docker-compose exec -e MIX_ENV=test application mix"
+alias dcs="docker-compose stop"
+alias dcu="docker-compose up -d"
 
 alias elixir-format="watchman -- trigger ./ format '**/*.ex' '**/*.exs' '**/*.default' -- mix format"
 
+function deamtw() {
+	docker-compose exec application mix test.watch
+}
```
Comment (nicholasjhenry) [inline civilcode.shrc:15]: Why does this need to be an alias?

File: civilcode.shrc [hunk 2]
```diff
@@ -4,9 +4,19 @@ alias dea="docker-compose exec application"
...
+function deamtwl() {
+	app="$1"; shift
+	docker-compose exec application mix cmd --app $app mix test.watch $*
+}
```
Comment (nicholasjhenry) [inline civilcode.shrc:25]: Trying to work out what the `l` stands for :-)

**Review by nicholasjhenry — COMMENTED** [review]
looks good -- just a couple of questions here

File: civilcode.shrc (hunk 1 above)
Comment (frahugo) [reply civilcode.shrc:15]: It does not. Lazy I guess. Will convert to an alias

File: civilcode.shrc (hunk 2 above)
Comment (frahugo) [reply civilcode.shrc:25]: `local`.. maybe `deamtwa` for `child app`?

**Comment by nicholasjhenry** [comment]
@frahugo How do you want to proceed with this?

**Pull request merged**